staff/didac-torres/spotify-template/mobile-template/index.html

```diff
@@ -0,0 +1,81 @@
+
+                
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=122" alt="album"></a><p class="albumtext">Temazos Chill</p></div>
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=123" alt="album"></a><p class="albumtext">Pop con Ñ</p></div>
+                
+                
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=124" alt="album"></a><p class="albumtext">POP</p></div>
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=125" alt="album"></a><p class="albumtext">Primavera Feliz</p></div>
+                
+                
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=126" alt="album"></a><p class="albumtext">Tapeo</p></div>
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=127" alt="album"></a><p class="albumtext">Deep House</p></div>
+                
+                
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=128" alt="album"></a><p class="albumtext">All Out 00s</p></div>
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=129" alt="album"></a><p class="albumtext">Top 50 España</p></div>
+                
+
+            </div>
+        </section>
+
+        <section>
+            <div class="dayquote">Géneros y estados de ánimo</div>
+            <div class="pool">
+                
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=12" alt="top8" "></a><p class="albumtext">GYM TIME</p></div>
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=13" alt="top8" "></a><p class="albumtext">POP</p></div>
+                
+                
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=14" alt="top8" "></a><p class="albumtext">HOUSE</p></div>
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=15" alt="top8" "></a><p class="albumtext">FIESTA</p></div>
+                
+                
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=16" alt="top8" "></a><p class="albumtext">ROMÁNTICO</p></div>
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=17" alt="top8" "></a><p class="albumtext">ALEGRÍA</p></div>
+                
+                
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=18" alt="top8" "></a><p class="albumtext">DANCE</p></div>
+                    <div class="album"><a href="#"><img src="https://picsum.photos/180?image=19" alt="top8" "></a><p class="albumtext">ROCK</p></div>
+                
+
+            </div>
+        </section>
+    </main>
+    <footer>
```

Comment:
@didactor91 in BEM, all this classes would be something like:
`footer__logo`, `footer__watch`,`footer__legal`.